tests/README.md

```diff
@@ -18,10 +18,20 @@ Travis (https://travis-ci.org/) is used to automatically build and run tests on
 
 Since Travis cannot `clasp login`, a `.clasprc.json` file is included that was created locally using `clasp login`.
 
-It is encrypted as `.clasprc.json.enc` and then decrypted by Travis:
+To then encrypt the `.clasprc.json` file, use these commands:
+
+```sh
+clasp login
+cp ~/.clasprc.json ./tests/.clasprc.json
+travis encrypt-file ./tests/.clasprc.json --add
+```
+
+This will add the following line to `.travis.yml`, which decrypts that file:
 
 ```openssl aes-256-cbc -K $encrypted_0f9bbf7a60f4_key -iv $encrypted_0f9bbf7a60f4_iv -in .clasprc.json.enc -out .clasprc.json -d || true```
 
+Make sure to commit the `./tests/.clasprc.json.enc` and __not__ `./tests/.clasprc.json`
```

Comment:
Can you remove this line and gitignore `.clasprc.json`?
Remove "and __not__ `./tests/.clasprc.json`"